src/hessenberg.jl

```diff
@@ -146,10 +146,7 @@ end
     
     n = size(S.data,1)
     
-    if n == 1
-        return Hessenberg(Matrix(S.data),Vector{eltype(S.data)}(undef,0),LA.UpperHessenberg(S.data),'L')
-    end
-
+    n == 1 && hessenberg!(S.data)
```

Comment:
Should this have a `return` statement?

Is this type stable?

Reply:
Why doesn't the subsequent code work if `n == 1`?

Reply:
It actually does, I just withdrew the line